jaxrs-api/src/main/java/jakarta/ws/rs/BadGatewayException.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2012, 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package jakarta.ws.rs;
+
+import jakarta.ws.rs.core.Response;
+
+/**
+ * A runtime exception indicating that the server, while acting as a gateway or proxy,
+ * received an invalid response from the upstream server it accessed in attempting to fulfill the request.
+ *
+ * @author Jeyvison Nascimento
```

Comment:
Adding your email address (like `Jeyvision Nascimento (jeyvision.nascimento@liferay.com)`) allows readers of your code to get in touch with you. :-)

Reply:
On it as soon as i figure out how to use my personal email in this PR  instead of liferay's one.

Reply:
@jeyvision You need to set up your local git config correctly and then rewrite the git history
See https://help.github.com/en/articles/setting-your-commit-email-address-in-git and https://stackoverflow.com/questions/3042437/how-to-change-the-commit-author-for-one-specific-commit/28845565.

Reply:
fixed